protoc-gen-openapiv2/main.go

```diff
@@ -113,6 +114,13 @@ func main() {
 		return
 	}
 
+	if *openAPIConfiguration != "" {
+		if err := reg.LoadOpenAPIConfigFromYAML(*openAPIConfiguration); err != nil {
+			emitError(err)
+			return
+		}
+	}
```

Comment:
One thing I was wondering was how this will interact with existing annotations in the file, I take it loading this before we go through the files means any file annotations take precedence? I think that's what we want, anyway.

Reply:
The extraction of OpenAPI options looks for the inline options first and looks up annotated registered from the YAML as a fallback (example:  [protoc-gen-openapiv2/internal/genopenapi/template.go:getMethodOpenAPIOption](https://github.com/grpc-ecosystem/grpc-gateway/pull/1665/files#diff-caf612205311291b1a186109ab244a23R1794)). It's easily changed to have the YAML options take precedence, and I think we should have the same precedence as the service annotations, though I confess to not knowing what that is.

Reply:
Yeah, I think this is fine.